src/components/DayPicker.jsx

```diff
@@ -244,6 +402,15 @@ export default class DayPicker extends React.Component {
     });
   }
 
+  isDayVisible(day, newMonth) {
+    const { numberOfMonths } = this.props;
+    const { currentMonth } = this.state;
+
+    const month = newMonth || currentMonth;
+    return !day.isBefore(month.clone().startOf('month')) &&
```

Comment:
not necessary to fix, but it might make this line a lot easier to read and understand if you pull out the `month.clone()....` pieces into their own `const`s